liquidctl/cli.py

```diff
@@ -28,13 +28,17 @@
   --time-off <value>          Time to wait with the LED turned off (seconds)
   --alert-threshold <number>  Threshold temperature for a visual alert (°C)
   --alert-color <color>       Color used by the visual high temperature alert
+  --direction <string>        If the pattern should move forward or backwords. [default: forward]
+  --start-led <number>        The first led to start the effect at (0 offset)
+  --num-leds <number>         The number of LED's the effect shoud apply to
```

Comment:
Perhaps `--maximum-leds` instead of `--num-leds`?  I would like to avoid abbreviations here.

Reply:
sure I can change it to that.